packages/blockchain-wallet-v4-frontend/src/modals/RequestBitcoin/SecondStep/selectors.js

```diff
@@ -6,6 +6,8 @@ export const getData = (state, ownProps) => {
   const amount = formValueSelector('requestBitcoin')(state, 'amount')
   const message = formValueSelector('requestBitcoin')(state, 'message')
   const satoshis = Exchange.convertBitcoinToBitcoin({ value: amount.coin, fromUnit: 'BTC', toUnit: 'SAT' }).value
-  const link = `https://blockchain.info/payment_request?address=${receiveAddress}&amount=${amount.coin}&message=${message}`
+  const payload = `bitcoin:${receiveAddress}?message=${message}&amount=${amount.coin}`
+  const encodedPayload = encodeURIComponent(payload)
+  const link = `https://blockchain.com/open/${encodedPayload}`
```

Comment:
The user flow should be:

1. generate payment request (https://blockchain.info/payment_request?address=${receiveAddress}&amount=${amount.coin}&message=${message})
2. user shares that payment request link with friend/whatever
3. friend/whatever gets https://login.blockchain.com/open/${encodedPayload} from blockchain.info/payment_request.

i.e the wallet should not generate the /open link, only blockchain.info/payment_request.

Also we should not hard code urls like this and instead get them from the root domain from wallet-options. See: https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4/src/redux/walletOptions/selectors.js#L5